configs/prod/roles/subscriptions.json

```diff
@@ -15,11 +15,11 @@
       ]
     },
     {
-      "name": "Subscriptions user",
+      "name": "Subscriptions viewer",
```

Comment:
please let the "name" same and add the "display_name" field as we did here https://github.com/RedHatInsights/rbac-config/pull/492